NewHorizons/Utility/ImageUtilities.cs

```diff
@@ -351,7 +351,16 @@ public static Texture2D MakeSolidColorTexture(int width, int height, Color color
             newTexture.Apply();
             return newTexture;
         }
-        
+
+        public static Sprite GetButtonSprite(JoystickButton button) => GetButtonSprite(ButtonPromptLibrary.SharedInstance.GetButtonTexture(button));
+        public static Sprite GetButtonSprite(KeyCode key) => GetButtonSprite(ButtonPromptLibrary.SharedInstance.GetButtonTexture(key));
+        private static Sprite GetButtonSprite(Texture2D texture)
+        {
+            var sprite = Sprite.Create(texture, new Rect(0, 0, texture.width, texture.height), new Vector2(0.5f, 0.5f), 100, 0, SpriteMeshType.FullRect, Vector4.zero, false);
```

Comment:
will this do a funny memory leak?

Reply:
I don't think creating a sprite will cause one. There is a few other things that create sprites (like vessel coordinate prompt, ship log star chart, ship log rumor mode, and ship log map mode.